composer.json

```diff
@@ -18,13 +18,13 @@
     "require": {
         "php": ">=7.4",
         "psr/log": "^1.0||^2.0||^3.0",
-        "symfony/yaml": "^3.4.0||^4.0.0||^5.0.0",
-        "symfony/config": "^3.4.0||^4.0.0||^5.0.0",
-        "symfony/console": "^3.4.0||^4.0.0||^5.0.0",
-        "symfony/filesystem": "^3.4.0||^4.0.0||^5.0.0",
-        "symfony/finder": "^3.4.0||^4.0.0||^5.0.0",
-        "symfony/translation": "^3.4.0||^4.0.0||^5.0.0",
-        "symfony/validator": "^3.4.0||^4.0.0||^5.0.0"
+        "symfony/yaml": "^3.4.0||^4.0.0||^5.0.0||^6.0.0",
+        "symfony/config": "^3.4.0||^4.0.0||^5.0.0||^6.0.0",
+        "symfony/console": "^3.4.0||^4.0.0||^5.0.0||^6.0.0",
+        "symfony/filesystem": "^3.4.0||^4.0.0||^5.0.0||^6.0.0",
+        "symfony/finder": "^3.4.0||^4.0.0||^5.0.0||^6.0.0",
+        "symfony/translation": "^3.4.0||^4.0.0||^5.0.0||^6.0.0",
+        "symfony/validator": "^3.4.0||^4.0.0||^5.0.0||^6.0.0"
```

Comment:
We are already not testing v3, so you can just as well remove that now.
Current CI says 4-6 if I see that correctly.

Reply:
I opened another PR to remove Symfony <4.4 support (see #1816).